dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -107,8 +107,14 @@ public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String d
                 Map.Entry<String, String> doiEntry = doiMap.entrySet().iterator().next();
                 String doi = doiEntry.getValue();
 
-                createAliasAndsetUpRelatedIdentifiers(depositMetadata, dockstoreGA4GHBaseUrl, dockstoreUrl,
-                        workflowUrl, workflow, workflowVersion, doi);
+                // Create the alias and make sure the alias is valid
+                // If it is not acceptable then an exception is generated
+                // in which case a deposition resource may be left on Zenodo
+                // that the user will have to clean up manually
+                doiAlias = createAliasUsingDoi(doi, user);
```

Comment:
This seems odd, which user is being passed in here?
It seems like the user is irrelevant here. 

I would have thought that regardless of whether a curator or an admin is requesting a zenodo DOI, there should not be a difference in behaviour. I can only rationalize a difference in behaviour for directly requesting DOIs by the endpoint as opposed to when requesting DOIs.

Reply:
Fixed; endpoint still checks user for privileges when decided on whether to allow special prefixes or forbidden Zenodo format
When creating a DOI the user privileges are not checked and only the invalid prefixes are forbidden